src/components/EventList/EventList.tsx

```diff
@@ -1,13 +1,37 @@
 "use client";
 
-import { FC } from "react";
+import { FC, useState } from "react";
 import Link from "next/link";
 import { formatDate } from "@/utils/formatDate";
 import { useQuery, gql, useMutation } from "@apollo/client";
 
 import type { IEvent } from "@/database/models/Event";
 
 import classes from "./EventList.module.css";
+import Popup from "../UI-kit/Popup/Popup";
+
+const Asdf: FC = () => {
+    const [showPopup, setShowPopup] = useState<boolean>(true);
```

Comment:
Тут возможно TS сам выведет тип из default value
```suggestion
    const [showPopup, setShowPopup] = useState(true);
```